### PR TITLE
Get Unit Tests and Lint Working

### PR DESCRIPTION
**Pull request by wrathchild317**
## Reviewers
@stefanagloginic 

Note: This is branching off of PR https://github.com/stefanagloginic/ValentinaListings/pull/3

## Goal 
- Setup Unit Tests With Mocha
- Fix linting error
- Add a pre-commit hook so it lints and runs unit tests before allowing commit

## Screenshots
**Lint**
pre_commit_lint
**Unit Tests**
unit_test_commit_hook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
-# ValentinaListings
```
Comment (stefanagloginic) [inline README.md]: why is this change showing up again? in this PR?

Comment (stefanagloginic) [reply README.md]: i think you will need to pull master after merging the other

File: components/Home.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default function Home() {
     return (
         <div>
             <h2>Home</h2>
-            <p>Welcome to the site!</p>
+            <p>Welcome to the Valentina's listings!</p>
```
Comment (stefanagloginic) [inline components/Home.js]: same

File: README.md (hunk 1 above)
Comment (wrathchild317) [reply README.md]: need to merge master into this

File: tests/unit/components/About.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/* globals describe,it */
+'use strict';
+
+import component from '../../../components/About';
+import {shallow} from 'enzyme';
+import React from 'react';
+import {expect} from 'chai';
+import {createMockComponentContext} from 'fluxible/utils';
+
+describe('About component tests', function () {
```
Comment (stefanagloginic) [inline tests/unit/components/About.js]: nice!

File: components/Home.js (hunk 2 above)
Comment (wrathchild317) [reply components/Home.js]: Mentioned in the note for the PR

**Comment by wrathchild317** [comment]
fix @stefanagloginic

**Pull request merged**